client/app/templates/Report/MoneyOut/_list.html.twig

```diff
@@ -55,7 +55,7 @@
     {% endif %}
 </dl>
 
-{% if (report.moneyOutExists == 'Yes') and (report.moneyTransactionsOut | length > 0) %}
```

Comment:
@MiaGordon91 Same as MoneyIn, It would be good if we could hide the Empty Label from showing whilst still showing the Add link when there are no items.